src/Fields/Media.php

```diff
@@ -102,7 +102,7 @@ public function setName($callback)
      */
     protected function fillAttributeFromRequest(NovaRequest $request, $requestAttribute, $model, $attribute)
     {
-    	$attr = request('__media__', []);
+        $attr = $request->__get('__media__', []);
```

Comment:
Hey @CristianGiordano, I'd suggest to use the array access instead of this method. There is no second argument also.

```php
$attr = $request['__media__'] ?? [];
```

Something like this maybe.